### DISCUSSION TITLE
synchronizacja zakupów

### DISCUSSION THREAD

**Discussion by pzebala** [allegro/allegro-api#7906] [Suggestions - rejected]
Brakuje w API procesów kupna. Dobrze  by było wprowadzić taką funkcjonalność. Sporo kupujemy, dobrze by było synchronizować to z ERPem.

**Comment by MaciejFrackowiak** [comment]
Sugestię zamykam jako duplikat już [istniejącej](https://github.com/allegro/allegro-api/discussions/5394) - tam znajdziesz bieżące aktualizacje.
